### ISSUE TITLE
About generating GFLOPs

### ISSUE THREAD

**Issue by CheungBH** [LeapLabTHU/AdaFocusV2#3]
Hello. Thanks for your excellent work.
I successfully ran your testing command with your pretrained model, which generated the mAP result **"79.0"**.
However, I didn't see the computational power GFLOPs result **"27.0"**. Are there any API to generate such a metric?

**Comment by yueyang2000** [comment]
You can use FLOPs counters such as `torchsummaryX` or `fvcore.nn.FlopCountAnalysis`.

**Comment by CheungBH** [comment]
Thank you for your reply,
I am trying to reproduce your result using your training command for activityNet

**CUDA_VISIBLE_DEVICES=0,1,2,3 python main_v2.py \
  --glance_arch mbv2 \
  --dataset actnet --data_dir PATH_TO_DATASET \
  --root_log LOG_DIR \
  --workers 8 --num_segments 16 --dropout 0.2  --fc_dropout 0.2  \
  --epochs 50 --lr 0.002 --lr_type cos --lr_steps 0 0 \
  --batch_size 8 --momentum 0.9 --weight_decay 1e-4 \
  --patch_size 128 --global_lr_ratio 0.5 --stn_lr_ratio 0.1 --classifier_lr_ratio 5.0 \
  --hidden_dim 1024 --stn_hidden_dim 2048 --sample**

However, the experiment can only achieve 59.24 mAP and 49.9 Prec@1, which is far less than the performance in the paper.
I wonder if there are some mistakes I made?

My arguments and log are attached.
[args.txt](https://github.com/LeapLabTHU/AdaFocusV2/files/10147746/args.txt)
[log.csv](https://github.com/LeapLabTHU/AdaFocusV2/files/10147747/log.csv)
